### ISSUE TITLE
Tornado uncaught exception when running example

### ISSUE THREAD

**Issue by rbharvs** [jbms/beancount-import#9]
Hi, I'm trying to run the `manually_entered` example (same thing happens for `fresh` example too), but can't find a way around this error which is repeatedly printed to the console:
```
ERROR:tornado.application:Uncaught exception GET /BEANCOUNT_IMPORT_SECRET_KEY_f31e30e228b14bf2e354100c5305bfdab40d608f/websocket (127.0.0.1)
HTTPServerRequest(protocol='http', host='localhost:8101', method='GET', uri='/BEANCOUNT_IMPORT_SECRET_KEY_f31e30e228b14bf2e354100c5305bfdab40d608f/websocket', version='HTTP/1.1', remote_ip='127.0.0.1')
Traceback (most recent call last):
  File "/Users/rbharvey/.virtualenvs/ledger/lib/python3.6/site-packages/tornado/websocket.py", line 952, in _accept_connection
    open_result = handler.open(*handler.open_args, **handler.open_kwargs)
  File "/Users/rbharvey/.virtualenvs/ledger/lib/python3.6/site-packages/beancount_import/webserver.py", line 267, in open
    self.set_nodelay(True)
  File "/Users/rbharvey/.virtualenvs/ledger/lib/python3.6/site-packages/tornado/websocket.py", line 561, in set_nodelay
    assert self.stream is not None
AssertionError
```
In the browser, it sort of flickers between what appears to be the web UI and the following error message: `Server connection closed, waiting to reconnect.`

**Comment by jbms** [comment]
This is an incompatibility with Tornado version 6, which was recently released.  I'll have to look into that, but in the meantime you can work around the problem with:

pip install 'torando==5'

**Comment by jbms** [comment]
This is fixed by #10
